ql/src/java/org/apache/hadoop/hive/ql/exec/ReplCopyTask.java

```diff
@@ -159,7 +159,7 @@ protected int execute(DriverContext driverContext) {
         if (!rwork.getListFilesOnOutputBehaviour(oneSrc)){
```

Comment:
I think it will go away. If required I can put a comment to explicitly check for the same in the load move to execution phase task ?